test/helpers.js

```diff
@@ -254,14 +254,18 @@ describe('helpers', function () {
     })
 
     describe('parameters', function () {
-      it('should not overwrite parameters from path when exists in operation', function () {
+      it('should add parameters from path to operation without overwriting', function () {
```

Comment:
@saharj I don't think that second part is covered in this PR
so...
```yaml
parameters:
  - name: josh
    in: query
    type: string
paths:
  /one:
    parameters:
    - name: sahar
      in: query
      type: string
    - name: josh
      in: query
      type: number
```

`/one.parameters` should have two parameters...
```yaml
- name: sahar
  in: query
  type: string
- name: josh
  in: query
  type: number
```